### PR TITLE
6219: TskCoreException thrown from normalizePhoneNum.

### PR DESCRIPTION
**Pull request by raman-bt**
Do not attempt to create an account when we do not have a valid phone number or email address.

### CODE REVIEW

File: bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java [hunk 1]
```diff
@@ -53,22 +53,69 @@ public static String normalizePhoneNum(String phoneNum) throws TskCoreException
...
+	/**
+	 * Checks if the given string is a valid phone number.
+	 *
+	 * @param phoneNum Phone number string to check.
+	 *
+	 * @return True if the given string is a valid phone number, false otherwise.
+	 */
+	public static boolean isValidPhoneNumber(String phoneNum) {
+		if (!StringUtils.isEmpty(phoneNum)) {
+			return phoneNum.matches("\\+?[0-9()\\-\\s]+");
```
Comment (rcordovano) [inline bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java:101]: Where did this validity check come from? Do we need to replace the code where this came from with a call to this method?

Comment (dannysmyda) [reply bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java:101]: Not the author, but that is from the normalizePhoneNumber method. I agree that this should call that method and catch the exception to ensure they don't diverge in their validation logic.

**Comment by dannysmyda** [comment]
@raman-bt Don't forget about addWebFormAddress() on the WebBrowserArtifactsHelper.

File: bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java (hunk 1 above)
Comment (dannysmyda) [reply bindings/java/src/org/sleuthkit/datamodel/CommunicationsUtils.java:101]: @raman-bt Could you change this via discussion above?

**Pull request merged**